deliverables/README.md

```diff
@@ -8,61 +11,134 @@ To answer questions, please just put the answer on the next line after the quest
+padding and margins 
+
+**What are the three different ways to include CSS in your project or use CSS to style a particular element?**  
+1. link the external styleet to your HTML<br/>
+2. write CSS in your HTML as in on-page style defintions  
+3. inline styles 
+
+
 
 ---
 ### Section 3: Git and Github  
-What is Git?  
-What is SCM?  
-What is a VCS?  
-Why is Git useful for a developer?  
-Why is Git useful for a team of developers?  
-How do you create a new Git repository for a project locally?  
-How do you create it on Github?  
-How do you commit changes?  
-What is the difference between staging and committing changes?  
-What is the difference between committing your changes and pushing them to Github?  
-What is the command to check the status of your current repo in git?  
-What is the command to see the history of your previous commits (from the command line)?  What are a few interesting variations (sets of options)?  
-How can you look through your historical commits on the Github website?  
-What is a "Merge"?  
-What is a "Pull Request"?  
-What is "Forking" a repo?  
-What is "Cloning" a repo?  
+**What is Git?**   
+A git is a distributed revision control system.
+
+**What is SCM?**  
+Source Code Management system
+
+**What is a VCS?**  
+Version Control System 
+
+**Why is Git useful for a developer?**   
+Git is useful becuase it can handle very large or small projects with efficiancy and speed.  
+
+**Why is Git useful for a team of developers?**   
+Git us useful for a team of developers because anyone can work on or edit the same files.
+
+**How do you create a new Git repository for a project locally?**  
+In Terminal change the working direct
+
+**How do you create it on Github?**  
+Click "new repository" name it, describe it, make it public or private, then initialize it with a README file, click create repository.  
+
+**How do you commit changes?**  
+in github you click "commit" locally, you have to use the terminal
+
+**What is the difference between staging and committing changes?**  
+Staging is like a preview of your modifications before you commit, you can view multiple documents and stage them then choose to commit them as the final step.
+
+**What is the difference between committing your changes and pushing them to Github?** 
+You have to commit them locally first then push them to Github
+
+**What is the command to check the status of your current repo in git?**   
+git status
+
+**What is the command to see the history of your previous commits (from the command line)?**  **What are a few interesting**
+**variations (sets of options)?**   
+git log, git diff, git merge, git show
+
+**How can you look through your historical commits on the Github website?**   
+click "history"
+
+**What is a "Merge"?**  
+A merge is combining your recent work into the master file
+
+**What is a "Pull Request"?**   
+Pull requests let you tell others about changes you've pushed to a repository on GitHub
```

Comment:
True, but also let's them know that they should merge your work into their repo or branch